src/modules/secureStorage.json

```diff
@@ -0,0 +1,265 @@
+{
+  "openrpc": "1.2.4",
+  "info": {
+    "title": "SecureStorage",
+    "version": "0.0.0",
+    "description": "A module for storing and retrieving secure data owned by the app"
+  },
+  "methods": [
+    {
+      "name": "get",
+      "summary": "Get stored value by key",
+      "tags": [
+        {
+          "name": "capabilities",
+          "x-uses": [
+            "xrn:firebolt:capability:secure:storage"
```

Comment:
maybe `xrn:firebolt:capability:storage:secure` as the later in the urn it becomes more specific. This is storage capability and the type of storage is our "secure" storage.